main.nf

```diff
@@ -674,26 +674,26 @@ process fastqc_trimmed {
  */
 if (!params.skip_adapter_trimming) {
     process porechop {
-        tag "$id"
+        tag "$name"
 
         input:
-        set id, file(lr) from files_long_raw
+        set val(name), file(lr) from ch_long_reads_raw
```

Comment:
Following the scheme of `ch_raw_reads_fastqc`, `ch_long_reads_raw` could be also named `ch_raw_long_reads`